taxonomy/utils.py

```diff
@@ -207,3 +207,23 @@ def get_blacklisted_course_skills(course_key, prefetch_skills=True):
     if prefetch_skills:
         qs = qs.select_related('skill')
     return qs.all()
+
+
+def get_course_jobs(course_key):
+    """
+    Get all the course jobs.
+
+    Arguments:
+        course_key (str): Key of the course whose course skills need to be returned.
+
+    Returns:
+        list: A list of all the course jobs.
+    """
+    course_jobs = set()
+    course_skills = get_whitelisted_course_skills(course_key)
+    for course_skill in course_skills:
+        job_skills = JobSkills.objects.select_related('skill', 'job').filter(skill=course_skill.skill)
```

Comment:
would it better to improve get all `JobSkills` in a single query? 
```
job_skills = JobSkills.objects.select_related('skill', 'job').filter(skill__in=[course_skill.skill for course_skill in course_skills])
```
and remove the outer for loop.